webui/apps/docs/src/content/docs/docs/quickstart.md

```diff
@@ -19,8 +19,9 @@ Welcome to the Eidolon Quickstart guide. This section covers environment setup,
 We know you are excited about creating your first agent, but first let's make sure we have everything we need to get started.
 
 ##### [OpenAI API Key](https://platform.openai.com/account/api-keys "Create an OpenAI key") 
-You should have an envrionment variable OPENAI_API_KEY set to your OpenAI API.
-Create a new key on [openai.com](https://platform.openai.com/api-keys) if you don't have one already.
+You should have an envrionment variable OPENAI_API_KEY associated with an account containing at least $5. set to your OpenAI API.
```

Comment:
typo: environment

"associated with a funded account (see the LLM provider for minimum funding requirements)"

remove leftover text: "... set to your OpenAI API."